File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_One/Cpp/program1.cpp

#include<iostream>


int main(){

	
	int num;

	printf("Enter the number\n");
	scanf("%d",&num);

	int flag =0;
	for(int itr =2 ; itr<=num/2 ;itr++){
		
		if(num%itr == 0){
			flag = 1;
			break;
		}	

	}	

	if(flag==0){
		printf("The number is Prime");
	}
	else{
		printf("The number is Not Prime");
	}	
}	

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/Java/program1.java

import java.io.*;

class Program {
	public static void main(String[] args) throws IOException{
			
		int n = 0, flag = 0;
		System.out.printf("Enter Number : ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		try{
			n = Integer.parseInt(br.readLine());
		}
		catch(NumberFormatException ne){
			System.exit(0);
		}

		if(n < 0)
			System.exit(0);
		for(int i = 2; i <= n/2; i++){
			
			if(n % i == 0 ){
				System.out.printf("Not prime Number\n");
				flag = 1;
				break;
			}
		}

		if(flag == 0){
			System.out.printf("Prime Number\n");
		}
	}
}